### PR TITLE
🇩🇪 German translation

### PR DESCRIPTION
**Pull request by Strubbl**
These commits translate strings to german

### CODE REVIEW

**Comment by Strubbl** [comment]
Hmm, i already fixed & pushed the comma bug introduced in the last commit, but in this pull request it does not get updated.

File: webtodotxt.py [hunk 1]
```diff
@@ -18,7 +18,8 @@
 app.config['LANGUAGES'] = {
     'en': 'English',
     'fr': 'Français',
-    'pt': 'Portuguese'
+    'pt': 'Portuguese',
+    'de': 'German'
```
Comment (Strubbl) [inline webtodotxt.py]: should it be "German" or in native language "Deutsch" here?

Comment (EpocDotFr) [reply webtodotxt.py]: Native language please.

**Comment by EpocDotFr** [comment]
Please add yourself in the contributor list [here](https://github.com/EpocDotFr/webtodotxt#contributors) in  the readme. Also describe in a few words what you have done.

**Comment by EpocDotFr** [comment]
Please also update the list of available languages along their code in the readme (in the Feature section).

**Pull request merged**